parm/config/gfs/config.base.j2

```diff
@@ -180,7 +180,7 @@ aero_fcst_runs="gdas" # When to run aerosol forecast: gdas, gfs, or both
 aero_anl_runs="gdas gfs" # When to run aerosol analysis: gdas, gfs, or both
 export DO_AERO_FCST="NO"
 export DO_AERO_ANL="NO"
-export DOBNDPNT_WAVE="NO"
+export DOBNDPNT_WAVE="YES"
```

Comment:
This will make this job run on all machines.  Just want to make sure it does run successfully everywhere.

Reply:
Given that the other point jobs run successfully everywhere, I don't know why this wouldn't.  That being said, I haven't tested this on every machine.  This is the change that I think would possibly require regression testing, we could always take this out to another PR if required.

Reply:
Moving this commit to: https://github.com/NOAA-EMC/global-workflow/pull/3670 which is likely a more appropriate place to test this anyways.